src/code/z_play_hireso.c

```diff
@@ -1,27 +1,1349 @@
+                } else {
+                    gDPSetPrimColor(gfx++, 0, 0, 155, 155, 155, 255);
+                }
+            } else {
+                if (CHECK_WEEKEVENTREG(
+                        sBombersNotebookEventColorWeekEventFlags[BOMBERS_NOTEBOOK_ENTRY_GET_EVENT(row, j) -
+                                                                 BOMBERS_NOTEBOOK_PERSON_MAX])) {
+                    gDPSetPrimColor(gfx++, 0, 0, sBombersNotebookEntryIconColors[eventIcon][0],
+                                    sBombersNotebookEntryIconColors[eventIcon][1],
+                                    sBombersNotebookEntryIconColors[eventIcon][2], 255);
+                } else {
+                    gDPSetPrimColor(gfx++, 0, 0, 155, 155, 155, 255);
+                }
+            }
+            BombersNotebook_TextureRectangle(&gfx, iconRectLeft * 4, (rectTop + yOffset) * 4,
+                                             (iconRectLeft + sBombersNotebookEventIconWidths[eventIcon]) * 4,
+                                             (rectTop + yOffset + sBombersNotebookEventIconHeights[eventIcon]) * 4, 0,
+                                             0, 0, 0x400, 0x400);
+            gDPSetPrimColor(gfx++, 0, 0, 255, 255, 255, 255);
+        } else {
+            unfinishedEvent = true;
+        }
+        j += BOMBERS_NOTEBOOK_ENTRY_SIZE;
+    }
+
+    *gfxP = gfx;
+}
+
+#define DEFINE_PERSON(_enum, photo, _description, _metMessage, _metFlag) &photo,
+
+TexturePtr sBombersNotebookPhotoTextures[] = {
+#include "tables/bombers_notebook/person_table.h"
+};
+
+#undef DEFINE_PERSON
+
+void BombersNotebook_DrawRows(BombersNotebook* this, Gfx** gfxP) {
+    static s16 sBarColorR = 0;
+    static s16 sBarColorG = 0;
+    static s16 sBarColorB = 0;
+    static s16 sBarColorTimer = 10;
+    static s16 sBarColorIndex = 0;
+    static s16 sBarColorTargetsR[2] = { 100, 0 };
+    static s16 sBarColorTargetsG[2] = { 80, 0 };
+    static s16 sBarColorTargetsB[2] = { 255, 0 };
+    Gfx* gfx = *gfxP;
+    s32 photoWidth;
+    s32 rowEnd;
+    s32 cursorPage;
+    u32 barRectLeft;
+    f32 photoRectDs;
+    u32 photoOffset;
+    s32 i;
+    u32 rectTop;
+    s16 colorStep;
+
+    colorStep = ABS_ALT(sBarColorR - sBarColorTargetsR[sBarColorIndex]) / sBarColorTimer;
+    if (sBarColorR >= sBarColorTargetsR[sBarColorIndex]) {
+        sBarColorR -= colorStep;
+    } else {
+        sBarColorR += colorStep;
+    }
+
+    colorStep = ABS_ALT(sBarColorG - sBarColorTargetsG[sBarColorIndex]) / sBarColorTimer;
+    if (sBarColorG >= sBarColorTargetsG[sBarColorIndex]) {
+        sBarColorG -= colorStep;
+    } else {
+        sBarColorG += colorStep;
+    }
+
+    colorStep = ABS_ALT(sBarColorB - sBarColorTargetsB[sBarColorIndex]) / sBarColorTimer;
+    if (sBarColorB >= sBarColorTargetsB[sBarColorIndex]) {
+        sBarColorB -= colorStep;
+    } else {
+        sBarColorB += colorStep;
+    }
```

Comment:
I can see this was probably the interface dev then. A landmark styling of theirs